aws-rds-dbclusterparametergroup/src/main/java/software/amazon/rds/dbclusterparametergroup/CallbackContext.java

```diff
@@ -16,8 +16,9 @@ public class CallbackContext extends StdCallbackContext implements TaggingContex
     private String dbClusterParameterGroupArn;
 
     private boolean parametersApplied;
-    private boolean clusterStabilized;
+    private boolean parameterGroupStabilized;
```

Comment:
Nit: just call it `stabilized`: it is clear from this context that we're talking about the owning resource.